src/components/ifxAccordion/ifxAccordion.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/Accordion',
+  component: 'ifx-accordion',
+  argTypes: {
+    amountOfItems: { control: 'number' },
```

Comment:
Title (caption) and Description (content) are currently not dynamic, and the user cannot change them.

Reply:
Only in Storybook. Do you think this is an issue?

Reply:
Do you want to set content for each tab within storybook? how should the controls look like?